### ISSUE TITLE
build with latest source, startup fail with gl exception

### ISSUE THREAD

**Issue by lokpro** [ikemen-engine/Ikemen-GO#644]
I have tried building the exe myself.
For each version I do: 1. git reset hard, 2. get.cmd, 3. build.cmd .

Found out, since this e7915ca9b6ba2b8370d751e9be0af682869cb10d verison, the program cannot startup.
I have successfully built and played older versions, so the data, external, etc folders should not be the problem.

<details>
  <summary>output (lastest version)</summary>

```
2022/08/15 03:58:03 Ikemen, GO!
2022/08/15 03:58:04 Using OpenGL 4.2.11672 Compatibility Profile Context (AMD Radeon HD 6570)
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0
signal arrived during external code execution

runtime.cgocall(0x16c3e40, 0xc0001494a8)
        runtime/cgocall.go:158 +0x4a fp=0xc000149480 sp=0xc000149448 pc=0x12c470a
github.com/go-gl/gl/v2.1/gl._Cfunc_glowObjectLabel(0x0, 0x82e2, 0x3, 0xffffffff, 0xc0001c85c0)
        _cgo_gotypes.go:31251 +0x52 fp=0xc0001494a8 sp=0xc000149480 pc=0x13e7f32
github.com/go-gl/gl/v2.1/gl.ObjectLabel(...)
        github.com/go-gl/gl@v0.0.0-20211210172815-726fda9656d6/v2.1/gl/package.go:24476
github.com/fyne-io/gl-js.ObjectLabel({0x1862578?, 0x16f7658?}, {0x17ab07a?, 0xb?})
        github.com/fyne-io/gl-js@v0.0.0-20220516203408-b35fbccb7063/gl_opengl.go:826 +0xc9 fp=0xc0001494f8 sp=0xc0001494a8 pc=0x1418d49
main.newShaderProgram({0x17cdad9?, 0x17b6069?}, {0x17ce340, 0x45f}, {0x17ab07a, 0xb})
        github.com/ikemen-engine/Ikemen-GO/src/gl.go:34 +0x8c fp=0xc000149538 sp=0xc0001494f8 pc=0x160b1cc
main.RenderInit()
        github.com/ikemen-engine/Ikemen-GO/src/render.go:102 +0x11e fp=0xc0001497c0 sp=0xc000149538 pc=0x164afbe
main.(*System).init(0x170c020, 0x18?, 0x0?)
        github.com/ikemen-engine/Ikemen-GO/src/system.go:496 +0x2b9 fp=0xc0001498f0 sp=0xc0001497c0 pc=0x168ff19
main.main()
        github.com/ikemen-engine/Ikemen-GO/src/main.go:89 +0x1f1 fp=0xc000149f80 sp=0xc0001498f0 pc=0x1648691
runtime.main()
        runtime/proc.go:250 +0x1fe fp=0xc000149fe0 sp=0xc000149f80 pc=0x12fb97e
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000149fe8 sp=0xc000149fe0 pc=0x1326b41

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:363 +0xd6 fp=0xc000037fb0 sp=0xc000037f90 pc=0x12fbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.forcegchelper()
        runtime/proc.go:302 +0xb1 fp=0xc000037fe0 sp=0xc000037fb0 pc=0x12fbbb1
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000037fe8 sp=0xc000037fe0 pc=0x1326b41
created by runtime.init.6
        runtime/proc.go:290 +0x25

goroutine 3 [GC sweep wait]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:363 +0xd6 fp=0xc000039f90 sp=0xc000039f70 pc=0x12fbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.bgsweep(0x0?)
        runtime/mgcsweep.go:278 +0x8e fp=0xc000039fc8 sp=0xc000039f90 pc=0x12e5f6e
runtime.gcenable.func1()
        runtime/mgc.go:178 +0x26 fp=0xc000039fe0 sp=0xc000039fc8 pc=0x12dab06
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000039fe8 sp=0xc000039fe0 pc=0x1326b41
created by runtime.gcenable
        runtime/mgc.go:178 +0x6b

goroutine 4 [GC scavenge wait]:
runtime.gopark(0xc00001e070?, 0x185f278?, 0x1?, 0x0?, 0x0?)
        runtime/proc.go:363 +0xd6 fp=0xc000047f70 sp=0xc000047f50 pc=0x12fbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.(*scavengerState).park(0x1a94d60)
        runtime/mgcscavenge.go:389 +0x53 fp=0xc000047fa0 sp=0xc000047f70 pc=0x12e3ff3
runtime.bgscavenge(0x0?)
        runtime/mgcscavenge.go:617 +0x45 fp=0xc000047fc8 sp=0xc000047fa0 pc=0x12e45e5
runtime.gcenable.func2()
        runtime/mgc.go:179 +0x26 fp=0xc000047fe0 sp=0xc000047fc8 pc=0x12daaa6
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000047fe8 sp=0xc000047fe0 pc=0x1326b41
created by runtime.gcenable
        runtime/mgc.go:179 +0xaa

goroutine 18 [finalizer wait]:
runtime.gopark(0x1a953a0?, 0x16e9aa0?, 0x0?, 0x0?, 0xc00003bf70?)
        runtime/proc.go:363 +0xd6 fp=0xc00003be28 sp=0xc00003be08 pc=0x12fbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.runfinq()
        runtime/mfinal.go:180 +0x10f fp=0xc00003bfe0 sp=0xc00003be28 pc=0x12d9c0f
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc00003bfe8 sp=0xc00003bfe0 pc=0x1326b41
created by runtime.createfing
        runtime/mfinal.go:157 +0x45
rax     0xc0001494a8
rbx     0xc0001494a8
rcx     0x82e2
rdi     0xc0001494a8
rsi     0x1a95040
rbp     0xc000149470
rsp     0x3ff6d8
r8      0xffffffff
r9      0xc0001c85c0
r10     0x0
r11     0x0
r12     0x185f238
r13     0x0
r14     0xc000034000
r15     0xffffffffffffffff
rip     0x0
rflags  0x10206
cs      0x33
fs      0x53
gs      0x2b
```
</details>

<details>
  <summary>output (e7915ca9b6ba2b8370d751e9be0af682869cb10d )</summary>

```
  2022/08/15 03:33:11 Ikemen, GO!
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0
signal arrived during external code execution

runtime.cgocall(0x169d720, 0xc0000755d8)
        runtime/cgocall.go:158 +0x4a fp=0xc0000755b0 sp=0xc000075578 pc=0x12a470a
github.com/go-gl/gl/v2.1/gl._Cfunc_glowObjectLabel(0x0, 0x82e2, 0x3, 0xffffffff, 0xc00009bc40)
        _cgo_gotypes.go:31251 +0x52 fp=0xc0000755d8 sp=0xc0000755b0 pc=0x13af672
github.com/go-gl/gl/v2.1/gl.ObjectLabel(...)
        github.com/go-gl/gl@v0.0.0-20211210172815-726fda9656d6/v2.1/gl/package.go:24476
github.com/fyne-io/gl-js.ObjectLabel({0x183ab30?, 0x16d1618?}, {0x1783bd2?, 0xb?})
        github.com/fyne-io/gl-js@v0.0.0-20220516203408-b35fbccb7063/gl_opengl.go:826 +0xc9 fp=0xc000075628 sp=0xc0000755d8 pc=0x13e02a9
main.RenderInit()
        github.com/ikemen-engine/Ikemen-GO/src/render.go:177 +0xa9 fp=0xc0000757c0 sp=0xc000075628 pc=0x1623729
main.(*System).init(0x16e45c0, 0x18?, 0x0?)
        github.com/ikemen-engine/Ikemen-GO/src/system.go:493 +0x2b9 fp=0xc0000758f0 sp=0xc0000757c0 pc=0x1669459
main.main()
        github.com/ikemen-engine/Ikemen-GO/src/main.go:88 +0x1f1 fp=0xc000075f80 sp=0xc0000758f0 pc=0x1620d91
runtime.main()
        runtime/proc.go:250 +0x1fe fp=0xc000075fe0 sp=0xc000075f80 pc=0x12db97e
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000075fe8 sp=0xc000075fe0 pc=0x1306b41

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:363 +0xd6 fp=0xc000037fb0 sp=0xc000037f90 pc=0x12dbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.forcegchelper()
        runtime/proc.go:302 +0xb1 fp=0xc000037fe0 sp=0xc000037fb0 pc=0x12dbbb1
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000037fe8 sp=0xc000037fe0 pc=0x1306b41
created by runtime.init.6
        runtime/proc.go:290 +0x25

goroutine 3 [GC sweep wait]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:363 +0xd6 fp=0xc000039f90 sp=0xc000039f70 pc=0x12dbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.bgsweep(0x0?)
        runtime/mgcsweep.go:278 +0x8e fp=0xc000039fc8 sp=0xc000039f90 pc=0x12c5f6e
runtime.gcenable.func1()
        runtime/mgc.go:178 +0x26 fp=0xc000039fe0 sp=0xc000039fc8 pc=0x12bab06
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000039fe8 sp=0xc000039fe0 pc=0x1306b41
created by runtime.gcenable
        runtime/mgc.go:178 +0x6b

goroutine 4 [GC scavenge wait]:
runtime.gopark(0xc00001e070?, 0x1837838?, 0x1?, 0x0?, 0x0?)
        runtime/proc.go:363 +0xd6 fp=0xc000045f70 sp=0xc000045f50 pc=0x12dbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.(*scavengerState).park(0x1a6ada0)
        runtime/mgcscavenge.go:389 +0x53 fp=0xc000045fa0 sp=0xc000045f70 pc=0x12c3ff3
runtime.bgscavenge(0x0?)
        runtime/mgcscavenge.go:617 +0x45 fp=0xc000045fc8 sp=0xc000045fa0 pc=0x12c45e5
runtime.gcenable.func2()
        runtime/mgc.go:179 +0x26 fp=0xc000045fe0 sp=0xc000045fc8 pc=0x12baaa6
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc000045fe8 sp=0xc000045fe0 pc=0x1306b41
created by runtime.gcenable
        runtime/mgc.go:179 +0xaa

goroutine 5 [finalizer wait]:
runtime.gopark(0x1a6b3e0?, 0x16c3a60?, 0x0?, 0x0?, 0xc00003bf70?)
        runtime/proc.go:363 +0xd6 fp=0xc00003be28 sp=0xc00003be08 pc=0x12dbd16
runtime.goparkunlock(...)
        runtime/proc.go:369
runtime.runfinq()
        runtime/mfinal.go:180 +0x10f fp=0xc00003bfe0 sp=0xc00003be28 pc=0x12b9c0f
runtime.goexit()
        runtime/asm_amd64.s:1594 +0x1 fp=0xc00003bfe8 sp=0xc00003bfe0 pc=0x1306b41
created by runtime.createfing
        runtime/mfinal.go:157 +0x45
rax     0xc0000755d8
rbx     0xc0000755d8
rcx     0x82e2
rdi     0xc0000755d8
rsi     0x1a6b080
rbp     0xc0000755a0
rsp     0x32faa8
r8      0xffffffff
r9      0xc00009bc40
r10     0x0
r11     0x0
r12     0x18377f8
r13     0x0
r14     0xc000034000
r15     0xffffffffffffffff
rip     0x0
rflags  0x10206
cs      0x33
fs      0x53
gs      0x2b
```
</details>

--
win7 64
go version go1.19 windows/amd64

**Comment by lokpro** [comment]
I tried the same built output on another PC. It works 
2022/08/22 16:01:57 Using OpenGL 4.5.13399 Compatibility Profile Context 15.200.
1065.0 (AMD RADEON HD 6670)

**Comment by Windblade-GR01** [comment]
It will be posible to give the specs of the 2 PCs were you tested?

**Comment by samhocevar** [comment]
It is a problem with OpenGL versions (4.2 vs. 4.5). I submitted a fix in #660.

**Comment by lokpro** [comment]
issue resolved. 
# 👍
